lib/http.js

```diff
@@ -202,8 +202,10 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       }
       const {profileAgentId} = req.params;
       const {profileAgent} = await profileAgents.get({id: profileAgentId});
-      const {capabilitySet} = await capabilitySets.get({profileAgentId});
-      const {zcaps: capabilities} = capabilitySet;
+
+      // all capabilities in the `zcaps` map will be delegated
+      const capabilities = Object.values(profileAgent.zcaps);
+
```

Comment:
I looked above and saw this was a GET API. This changes state and needs to become a POST API instead -- and not just for that reason which is good enough on its own, but also for potential security issues with CSRF and so on.

Reply:
Done.